language_load.py

```diff
@@ -1,99 +1,152 @@
 #!/usr/bin/env python3
+"""
+Imports translations into OpenRCT2\'s JSON objects given a language and an input file
```

Comment:
The apostrophe doesn't need to be escaped in a Python docstring, right?
```suggestion
Imports translations into OpenRCT2's JSON objects given a language and an input file
```